packages/oscal-react-library/src/components/OSCALControlProse.js

```diff
@@ -315,27 +308,27 @@ export function OSCALReplacedProseWithParameterLabel(props) {
 
   if (!props.isImplemented) {
     return (
-      <NotImplementedStatement>
+      <NotSpecifiedTypography>
```

Comment:
Something like this does not work. 

```suggestion
      <NotSpecifiedTypography component={div}>
```